scalableminds/webknossos
# Store Uploader ID with Dataset

### Steps to test:
- Upload a dataset via UI
- `select _uploader from datasets` should show uploader id
- Datasets reported from disks should have the value null in that column.
- Dataset json representation (e.g. in GET /api/datasets`) should not change.

### Issues:
- fixes #4987 

------
